src/main/java/org/folio/edge/oaipmh/OaiPmhHandler.java

```diff
@@ -251,4 +283,45 @@ private void handleNotAcceptableError(RoutingContext ctx, HttpServerRequest requ
     notAcceptable(ctx,
         "Accept header must be \"text/xml\" for this request, but it is " + "\"" + unsupportedType + "\"" + ", can not send */*");
   }
+
+  /**
+   * This method make request to mod-configuration module and receive config setting associate with name behavior
+   *
+   * @param ctx routing context
+   * @return value of field errorsProcessing
+   */
+  private CompletableFuture<String> getErrorsProcessingConfigSetting(RoutingContext ctx) {
+    String QUERY = "module==OAIPMH and configName==behavior";
+    CompletableFuture<String> future = new VertxCompletableFuture<>();
+    String tenantId = ctx.request().headers().get(X_OKAPI_TENANT);
+    String token = ctx.request().headers().get(X_OKAPI_TOKEN);
+    String okapiURL = ctx.request().headers().get(X_OKAPI_URL);
+    try {
+      new ConfigurationsClient(okapiURL, tenantId, token).getConfigurationsEntries(QUERY, 0, 100, null, null,
+          responseConfig -> responseConfig.bodyHandler(body -> {
+            try {
+              if (responseConfig.statusCode() != 200) {
+                logger.error("Error getting configuration. Expected status code 200 but was: " + responseConfig.statusCode());
+                return;
```

Comment:
Complete future with error response here as well. 
Something like that.
future.complete(buildFailureResponse())

private Response buildFailureRsponse(String error msg){
...build reponse object
return response;
}

Reply:
I return a completed future with error code